pkg/controller/sidecarset/sidecarset_processor.go

```diff
@@ -163,7 +163,8 @@ func (p *Processor) updatePods(control sidecarcontrol.SidecarControl, pods []*co
 			klog.Errorf("update NotUpgradable PodCondition error, s:%s, pod:%s, err:%v", sidecarset.Name, pod.Name, err)
 			return err
 		}
-		sidecarcontrol.UpdateExpectations.ExpectUpdated(sidecarset.Name, sidecarcontrol.GetSidecarSetRevision(sidecarset), pod)
+		// Since the pod sidecarSet hash is not updated here, it cannot be called ExpectUpdated
```

Comment:
just remove the code? we only comment code that will be uncommented in the future

Reply:
// TODO: add ResourceVersionExpectation instead of UpdateExpectations